<ui version="4.0" >
 <class>ColorFontPage</class>
 <widget class="QWidget" name="ColorFontPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>512</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Color and Font Settings</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Default Font</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="QLabel" name="label_17" >
        <property name="text" >
         <string>Server Ouput</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="serverOutputButton" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QLabel" name="label_18" >
        <property name="text" >
         <string>Input Line</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QPushButton" name="inputLineButton" >
        <property name="text" >
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QGroupBox" name="groupBox_4" >
     <property name="title" >
      <string>Default Colors</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_19" >
        <property name="text" >
         <string>Foreground</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="foregroundButton" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_20" >
        <property name="text" >
         <string>Background</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="backgroundButton" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>Dark Colors</string>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Black</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QPushButton" name="blackButton" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Red</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QPushButton" name="redButton" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Green</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QPushButton" name="greenButton" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Yellow</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QPushButton" name="yellowButton" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>Blue</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" >
         <widget class="QPushButton" name="blueButton" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="0" >
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>Magenta</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" >
         <widget class="QPushButton" name="magentaButton" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="0" >
         <widget class="QLabel" name="label_7" >
          <property name="text" >
           <string>Cyan</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1" >
         <widget class="QPushButton" name="cyanButton" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="0" >
         <widget class="QLabel" name="label_8" >
          <property name="text" >
           <string>Gray</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1" >
         <widget class="QPushButton" name="grayButton" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Bright Colors</string>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_11" >
          <property name="text" >
           <string>Dark gray</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QPushButton" name="darkGrayButton" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_14" >
          <property name="text" >
           <string>Bright red</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QPushButton" name="brightRedButton" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_16" >
          <property name="text" >
           <string>Bright green</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QPushButton" name="brightGreenButton" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="label_10" >
          <property name="text" >
           <string>Bright yellow</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QPushButton" name="brightYellowButton" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="label_9" >
          <property name="text" >
           <string>Bright blue</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" >
         <widget class="QPushButton" name="brightBlueButton" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="0" >
         <widget class="QLabel" name="label_13" >
          <property name="text" >
           <string>Bright magenta</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" >
         <widget class="QPushButton" name="brightMagentaButton" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="0" >
         <widget class="QLabel" name="label_12" >
          <property name="text" >
           <string>Bright cyan</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1" >
         <widget class="QPushButton" name="brightCyanButton" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="0" >
         <widget class="QLabel" name="label_15" >
          <property name="text" >
           <string>White</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1" >
         <widget class="QPushButton" name="whiteButton" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
